Makefile

```diff
@@ -68,7 +68,7 @@ oci-image-tool:
 
 schema-fs:
 	@echo "generating schema fs"
-	@cd schema && echo -e "$$(cat ../.header)\n\n$$(go generate)" > fs.go
+	@cd schema && echo "$$(cat ../.header)\n\n$$(go generate)" > fs.go
```

Comment:
On Fri, Jun 03, 2016 at 12:06:45PM -0700, Stephen Day wrote:
> > @@ -68,7 +68,7 @@ oci-image-tool:
> >  
> >  schema-fs:
> >  	@echo "generating schema fs"
> > -	@cd schema && echo -e "$$(cat ../.header)\n\n$$(go generate)" > fs.go
> > +	@cd schema && echo "$$(cat ../.header)\n\n$$(go generate)" > fs.go
> 
> You're welcome to submit a PR.

Sure.  Until I do (tonight?), can we drop the drive-by change here?